qa-pipelines/qa-pipeline.yml.erb

```diff
@@ -142,11 +179,15 @@ jobs:
     - put: pool.kube-hosts
       params: {acquire: true}
       timeout: 2m
+    <% if status_reporting %>
     - task: cf-get-commit-id
       file: ci/qa-pipelines/tasks/cf-get-commit-id.yml
+      <% if is_upgrade_pipeline %>
+      params:
+        CAP_BUNDLE_URL: <%= bundle_url_from_version(enable_cf_deploy_pre_upgrade) %>
```

Comment:
Another get-commit-id step which gets the post-upgrade commit id runs just before the upgrade task step, so it should be fine, though this will set a status on the github commit for the pre-upgrade version which will never get updated unless there's a failure in the pre-upgrade portion. This could potentially be a bit confusing, but not really a problem as we're generally testing the version we're upgrading to

Reply:
ok. we mark this as a known bug and move on.

Reply:
https://jira.suse.com/browse/CAP-620